request/models.py

```diff
@@ -21,6 +21,7 @@ class Request(models.Model):
     method = models.CharField(_('method'), default='GET', max_length=7)
     path = models.CharField(_('path'), max_length=255)
     time = models.DateTimeField(_('time'), default=timezone.now, db_index=True)
+    response_time = models.FloatField(_('response_time'), null=True)
```

Comment:
Could [`models.DurationField`](https://docs.djangoproject.com/en/5.0/ref/models/fields/#durationfield) be more appropriate? One obvious advantage I could see is that the type is self-documenting that the field stores what unit. Whereas "response time" and "float" isn't clear if this is miliseconds, seconds, nanoseconds etc in units.

Reply:
This is a good point.  I'll change it to DurationField!